spec/system/support_interface/countries_spec.rb

```diff
@@ -59,18 +59,34 @@
   private
 
   def given_countries_exist
-    create(:region, :national, country: create(:country, code: "IE"))
-    create(:region, :national, country: create(:country, code: "PL"))
-    united_states = create(:country, code: "US")
+    create(
+      :region,
+      :national,
+      country: create(:country, code: "IE", subject_limited: false),
```

Comment:
```suggestion
      country: create(:country, code: "IE"),
```

Do we need to specify this field directly if it defaults to `false`?